src/gui/systray.h

```diff
@@ -82,18 +84,13 @@ class Systray
     void showMessage(const QString &title, const QString &message, MessageIcon icon = Information);
     void showUpdateMessage(const QString &title, const QString &message, const QUrl &webUrl);
     void setToolTip(const QString &tip);
-    bool isOpen();
+
     QString windowTitle() const;
     bool useNormalWindow() const;
-    void createCallDialog(const Activity &callNotification);
+    bool syncIsPaused() const;
+    bool isOpen() const;
```

Comment:
speaking of improving the API, you may want to also add `Q_REQUIRED_RESULT`
you want this on every method that are const and return a value
if they are const the only reason to call them is to get the returned value so requiring that it is used makes sense

Reply:
Okay, thanks, added `Q_REQUIRED_RESULT`